MoltenVK/MoltenVK/GPUObjects/MVKRenderPass.mm

```diff
@@ -98,7 +99,7 @@
 			}
 
 		} else {
-			caCnt--;
+			caUsedCnt--;
```

Comment:
One more change. Sorry...I should have thought of this on my first review. 🤦🏻‍♂️

Now that you have a separate variable for tracking this...can we make `caUsedCnt` just start at zero...increment it within the `if()` block...and remove the `else` block entirely. Seems to be a more logical direction.

Reply:
Done.

Reply:
Thanks!